ribs/emitters/_iso_line_emitter.py

```diff
@@ -151,15 +158,9 @@ def ask(self):
 
         if self.archive.empty:
             solution_batch = np.expand_dims(self._x0, axis=0) + iso_gaussian
+            return np.clip(solution_batch, self.lower_bounds, self.upper_bounds)
         else:
             parents = self.archive.sample_elites(self._batch_size)["solution"]
```

Comment:
Just use one call to `sample_elites` here with 2 * self._batch_size, then reshape it. It's also inefficient to concatenate the two arrays.